prod/freestanding/logs/default.env

```diff
@@ -0,0 +1,23 @@
+# Default docker-compose environment file (.env)
+# https://docs.docker.com/compose/environment-variables/#the-env-file
+# environmental variables for interpolation in docker-compose YAML files
+
+# BASE_DOMAIN=localtest.me
+
+# https://docs.docker.com/compose/reference/envvars/#compose_project_name
+# Containers started with the below value will have their names prefixed with it
+# Required on shared infrastructure
+# COMPOSE_PROJECT_NAME=logserver-dev
+
+COMPOSE_FILE=../../../base/freestanding/logs/logserver/docker-compose.yaml:../../../base/freestanding/logs/docker-compose.ingress.yaml
+
+
+PGRST_JWT_SECRET=
+
+OIDC_PROXY_SECRET=
+# must match __KEYCLOAK_FEMR_REALM_LOGSERVER_OPENID_CLIENT_SECRET
```

Comment:
smooshed up w/o whitespace, i *assume* this comment applies to the line below?  but that doesn't seem right...